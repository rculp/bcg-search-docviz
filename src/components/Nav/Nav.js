import React from 'react';
import { Link } from 'react-router-dom';
import { UI_URL } from 'config';
import Menu from 'components/Menu/Menu';
import './Nav.scss';

const Nav = () => (
  <nav>
    <Menu>
      <Link to={UI_URL.HOME}>
        <div className="logo">M</div>
      </Link>
    </Menu>
  </nav>
);

export default Nav;
