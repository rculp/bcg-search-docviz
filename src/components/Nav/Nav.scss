@import 'src/global-styles/colors';
@import 'src/global-styles/responsive';

nav {
  height: 100%;

  .ui.menu {
    height: 100%;
    border: 0;
  }

  .logo {
    height: 100%;
    background-color: $bcg-green--dark;

    display: flex;
    justify-content: center;
    align-items: center;

    color: $white;
    font-family: 'BCG-Henderson-Sans', sans-serif;

    width: 56px;
    font-size: 2.8rem;
    @media (max-width: map-get($grid-breakpoints, sm)) {
      width: 48px;
      font-size: 2.2rem;
    }
  }
}
