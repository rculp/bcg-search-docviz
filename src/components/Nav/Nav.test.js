import { shallow } from 'enzyme';

import React from 'react';

import Nav from './Nav';

describe('Nav', () => {
  it('renders and matches our snapshot', () => {
    const component = shallow(
      <Nav />
    );
    expect(component).toMatchSnapshot();
  });
});
