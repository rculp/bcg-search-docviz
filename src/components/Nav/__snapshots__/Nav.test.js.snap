// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders and matches our snapshot 1`] = `
<nav>
  <Menu>
    <Link
      replace={false}
      to="/?q="
    >
      <div
        className="logo"
      >
        M
      </div>
    </Link>
  </Menu>
</nav>
`;
